swap_meet/clothing.py

```diff
@@ -1,2 +1,11 @@
-class Clothing:
-    pass
\ No newline at end of file
+from swap_meet.item import Item 
+
+
+class Clothing(Item):
+
+    def __init__(self, id=None, condition=0, fabric="Unknown"):
```

Comment:
[opinion] I prefer to put space around both sides of `=` like: `fabric = "Unknown"`